spec/requests/api/issues_spec.rb

```diff
@@ -241,6 +248,14 @@
       json_response['title'].should == 'updated title'
     end
 
+    it "should update a project issue by iid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.